raven/backend.go

```diff
@@ -88,6 +129,9 @@ func fromGlogEvent(e glog.Event) *Event {
 		}
```

Comment:
Sorry, I bet this is annoying, but what do you think about adding a case to the above switch statement? No need to put it through a generic map

```go

type altDsn string

func DsnData(dsn string) glog.Data {
  return altDsn(dsn)
}

...
switch t := d.(type) {
case altDsn:
  eve.TargetDsn = string(altDsn)
...
```